File: computer_science/99problems/javascript/1.01.js
let list = ['a', 'b', 'c', 'd'];

function ttyFindLast(list) {
    let length = list.length;

    if (length == 0) {
      console.log("This list is empty.")
  } else {
      console.log(list[length - 1])
  }
}

ttyFindLast(list)
// => d

File: computer_science/99problems/python/1.01.py
list = ['a', 'b', 'c', 'd']

def tty_find_last(list):
    length = len(list)

    if length == 0:
        print("This list is empty.")
    else:
        print(list[length - 1])

tty_find_last(list)
# => d
